teacher.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+require_relative 'person'
+# Class Teacher inherits from Person
+class Teachear < Person
+  def initialize(specialization, age)
```

Comment:
- Hey, Please, kindly add the name attribute as a person either [Teacher or Student] should and must have a name. please do add the name attribute.
- Also, do add the parrent_permission to be sure they are not underage to access the resources.